converter/datamodel.py

```diff
@@ -436,13 +436,44 @@ def from_magetab(cls, name, datafile_objects, file_attributes):
             data_type = None
 
         comments = common_file_attributes.get("comments")
+        print(comments)
```

Comment:
Should this be here, or was it a test line?

Reply:
Yep, just debugging... The whole project is still in the debugging phase, I am going to leave it for now.